src/shared_utilities/drivers.jl

```diff
@@ -79,7 +80,7 @@ struct PrescribedAtmosphere{FT, LP, SP, TA, UA, QA, RA, CA, DT} <:
         ref_time,
         h::FT;
         gustiness = FT(1),
-        c_co2 = nothing,
+        c_co2 = (t) -> 4.2e-4,
```

Comment:
instead of the default being `nothing`, I set it to return the current co2 concentration.